internal/controller/dependencies.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 
 	"github.com/hashicorp/go-multierror"
+	"golang.org/x/exp/maps"
```

Comment:
kinda off topic but are we planning to move to go 1.21 soon? the `maps` and `slices` package are part of the std lib as of that release

Reply:
No idea -- I assume there's a standard cadence that different teams follow @ Hashi